src/components/location/helpers.ts

```diff
@@ -0,0 +1,7 @@
+export const timezone = (offset: number | undefined): string | null => {
```

Comment:
nadal nie mam pojęcia co ta funkcja zwraca, ale wydaje mi się ze nie jest to timezone

Reply:
funkcja zwraca różnicę od UTC - zmieniłem na `getUTCTimeDifference`